src/main/resources/resources.properties

```diff
@@ -201,7 +201,7 @@ ui.BannerView.courseEnded=The course has ended.
 ## Tutorial
 ui.tutorial.TutorialAction.confirmStart=A tutorial will start.
 ui.tutorial.TutorialAction.confirmCancel=Cancel tutorial?
-ui.tutorial.TutorialAction.end=The tutorial has ended.
+ui.tutorial.TutorialAction.end=<html>The tutorial has ended. Would you like to submit your result to the A+ system?<br>If you choose not to submit the result now, you will have to do the tutorial again.</html>
```

Comment:
Ideally, student should be able to choose to send it later but if that's difficult to implement, this goes for now. Let's discuss in the weekly with the stakeholders.